public/app/features/auth-config/fields.tsx

```diff
@@ -131,6 +113,132 @@ export const sectionFields: Section = {
       ],
     },
   ],
+  google: [
+    {
+      name: 'General settings',
+      id: 'general',
+      fields: ['name', 'clientId', 'clientSecret', 'scopes', 'allowSignUp', 'autoLogin', 'signoutRedirectUrl'],
+    },
+    {
+      name: 'User mapping',
+      id: 'user',
+      fields: ['allowAssignGrafanaAdmin', 'roleAttributePath', 'roleAttributeStrict', 'skipOrgRoleSync'],
```

Comment:
Please fix the order of these. The order should be (based on generic_oauth):  'roleAttributePath', 'roleAttributeStrict','allowAssignGrafanaAdmin','skipOrgRoleSync'.

Reply:
Fixed this for all providers.